File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/MedicinalProductUndesirableEffect.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class MedicinalProductUndesirableEffect : DomainResource, IResource
{
    public Base.ResourceReference[]? Subject { get; set; }
    public Base.Population[]? Population { get; set; }
    public Base.CodeableConcept? Classification { get; set; }
    public Base.CodeableConcept? FrequencyOfOccurrence { get; set; }
    public Base.CodeableConcept? SymptomConditionEffect { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/MedicinalProductUndesirableEffect.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import Population
from base import DomainResource
from base import Reference

@dataclass(kw_only=True)
class MedicinalProductUndesirableEffect(DomainResource):
    subject: Optional[List[Reference]] = field(default_factory=list)
    population: Optional[List[Population]] = field(default_factory=list)
    classification: Optional[CodeableConcept] = None
    frequency_of_occurrence: Optional[CodeableConcept] = None
    symptom_condition_effect: Optional[CodeableConcept] = None